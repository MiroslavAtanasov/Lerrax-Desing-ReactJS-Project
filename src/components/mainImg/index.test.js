import React from 'react'
import renderer from 'react-test-renderer'
import MainImg from '.'
import TestingEnvironment from '../../test-utils/router'

jest.mock('../dots', () => 'Dots');


describe('MainImg Component', () => {
    it('should render MainImg component', () => {
        const tree = renderer.create(
            <TestingEnvironment value={{
                user: {
                    loggedIn: true,
                    id: '123'
                }
            }}>
                <MainImg />
            </TestingEnvironment>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
})