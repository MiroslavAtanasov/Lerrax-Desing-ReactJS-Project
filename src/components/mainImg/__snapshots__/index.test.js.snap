// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainImg Component should render MainImg component 1`] = `
<div
  className="container"
>
  <img
    alt="mainImage"
    className="mainImg"
    onClick={[Function]}
    src="Feb-20-marquees1 placeholdr.jpg"
  />
  <div
    className="inner-container"
  >
    <Dots
      onClick={[Function]}
    />
    <Dots
      onClick={[Function]}
    />
    <Dots
      onClick={[Function]}
    />
    <Dots
      onClick={[Function]}
    />
  </div>
</div>
`;
